languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/arabic_tags.txt

```diff
@@ -0,0 +1,188 @@
+Description of Arabic tag sets
+=============================
+The arabic tag sytax for grammar check is composed of 4 parts
+
+The tag starts by word class
+for example 
+V: Verb
+N: Noun
+P: tool or partical
+
+Nouns
+======
+
+3 parts:
+======
+* word class and category
+* Conjugation ( suffixes used to conjugate words in dual form, feminin form).
+* Affixes ( prefixes and encletics)
+
+
+
+Word class and category:
+======
+ define the attributes of word in dictionary, for example,
+contains 3 parts
+- Word type:
+	N: noun,
+	V: verb,
+	P: particle
+- Word sub class:
+	M	Masdar	مصدر
+	A	Adjective	صفة
+	A	Adjective similair	صفة مشبهة
+	A	Adjective superlative	صيغة مبالغة
+	A	maf3oul	اسم مفعول
+	A	fa3il	اسم فاعل
+	P	Proper Noun	اسم علم
+	S	Superlative	اسم تفضيل
+	J	jamid	جامد
+- Not used
+
+
+Noun conjugation  : 4 letters
+======
+	- Gender suffix:
+		M: masculine ( ون is a mark of masculine)
+		F: feminine (ة or ات)
+		-: none
+	- Number suffix:
+		1: single
+		2: dual
+		3: plural ( regular masculine or feminine plural, or irregular plural)
+		3I: irregular plural form
+	- Case ( إعراب)
+		U	marfou3	مرفوع
+		0	manjzoum	 مجزوم
+		I	majrour	مجرور
+		A	mansoub	منصوب
+		B	mabni	مبني
+		-	undef	لاشيء
+	- Inflection Mark علامة الإعراب
+		u	damma	الضمة
+		a	fatha	الفتحة
+		i	kasra	الكسرة
+		0	sukun	السكون
+		U	dammatan	تنوين الضمة
+		e	fathatan	تنوين الفتحة
+		E	alef_fathatan	تنوين الألف
+		I	kasratan	تنوين الكسرة
+		A	alef	الألف
+		W	waw	الواو
+		Y	yeh	الياء
+		N	noon	ثبوت النون
+		-	undef	لاشيء
+
+Noun prefixes and encletic :
+=============
+contains 3 parts
+W: conjonction: starts by WAW or FEH, take 3 values: W: for waw, F; for Feh, -: none.
+B: preposition: B: for Beh, K: for Kaf, L: for Lam, by default we use B, -: none.
+L/H: definite article, L: for AL,  H: attached pronouns  -: none.
+	H : define the extended words added to the lexem: الضمائر المضافة
+	H: if have encletic
+
+Example:
+شرفك	شرف	NS-;M3U-;--H
+شرفي	شرف	NS-;M3U-;---
+الشرف	شرف	NS-;M3A-;--L
+شرفتك	شرف	NS-;F3A-;--H
+شرف	شرف	NS-;M3U-;---
+شرفين	شرف	NS-;M3I-;---
+بالشرف	شرف	NS-;M3I-;-BL
+الشرفي	شرف	NS-;M3A-;--L
+شرف	شرف	NS-;M3A-;---
+شرفي	شرف	NS-;M3A-;---
+
+
+
+
+
+
+Verbs
+=====
+Word class + conjugation + affixes
+
+verb Word class  :
+=============
```

Comment:
Underline whole section title. 
Remove space before colon or preferably remove the colon